tests/ttnn/unit_tests/test_unsqueeze.py

```diff
@@ -10,20 +10,24 @@
 
 
 @pytest.mark.parametrize(
-    "input_shape, dim",
+    "input_shape, dim, layout",
```

Comment:
Please add a few tests with both last dims > 32 but not divisible by tile.

Reply:
Added them